modules/metrics/collector.go

```diff
@@ -249,6 +255,14 @@ func (c Collector) Collect(ch chan<- prometheus.Metric) {
 		prometheus.GaugeValue,
 		float64(stats.Counter.Issue),
 	)
+	for _, il := range stats.Counter.IssueByLabel {
```

Comment:
How is `nil` handled by `range`?
Will the loop simply never be executed, or does it throw an `NPE`?
Because if it is the latter, you should wrap it inside an if as it can be `nil` when `ENABLED_ISSUE_BY_LABEL = false`.
If it simply never executes the loop, then ignore this comment.

Reply:
nil by range is handled, it just wont runn the